src/ecrc-api/src/test/java/ca/bc/gov/open/ecrc/controller/UpdateServiceFinancialTxnControllerTest.java

```diff
@@ -0,0 +1,64 @@
+package ca.bc.gov.open.ecrc.controller;
+
+import ca.bc.gov.open.ecrc.controller.UpdateServiceFinancialTxnController;
+import ca.bc.gov.open.ecrc.EcrcServicesImpl;
+import ca.bc.gov.open.ecrc.exception.EcrcExceptionConstants;
+import ca.bc.gov.open.ecrc.exception.EcrcServiceException;
+import ca.bc.gov.open.ecrc.model.RequestNewCRCService;
+import ca.bc.gov.open.ecrc.model.RequestUpdateServiceFinancialTxn;
+import ca.bc.gov.open.ecrc.objects.UpdateServiceFinancialTxn;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class UpdateServiceFinancialTxnControllerTest {
+    @InjectMocks
+    UpdateServiceFinancialTxnController updateServiceFinancialTxnController;
+
+    @Mock
+    EcrcServicesImpl ecrcServices;
+
+    RequestUpdateServiceFinancialTxn updateServiceFinancialTxn;
+
+    @BeforeEach
+    public void initMocks(){
+        MockitoAnnotations.initMocks(this);
+    }
+    @DisplayName("Success - updateServiceFinancialTxn controller")
+    @Test
+    public void testSuccessfulCreate() throws EcrcServiceException {
+        updateServiceFinancialTxn = new RequestUpdateServiceFinancialTxn();
+        Mockito.when(ecrcServices.updateServiceFinancialTxn(updateServiceFinancialTxn)).thenReturn(new ResponseEntity<>("SOMESTRING", HttpStatus.OK));
+        ResponseEntity<String> result = updateServiceFinancialTxnController.updateServiceFinancialTxn(updateServiceFinancialTxn);
+        Assertions.assertEquals("SOMESTRING", result.getBody());
```

Comment:
should we be checking for HttpStatus.OK for consitency?

Reply:
Good call, I have added it.